_timeline/2017-12-29-34th-Chaos-Communication-Congress.md

```diff
@@ -0,0 +1,5 @@
+---
+title: 34th Chaos Communication Congress
+date: 2017-12-29
+---
+A couple of us from Toronto Mesh went to Leipzig and presented [DIY Networks in Toronto](https://media.freifunk.net/v/diy-networks-in-toronto) at the [Freifunk](https://freifunk.net) Open Infrastructure Stage, where community network enthusiasts gathered at [34C3](https://events.ccc.de/congress/2017/wiki).
```

Comment:
Maybe tighten the language to something like "Members went to 34c3 in Leipzig and a few of us presented on..."? (technically more than a few went ;)